src/screens/CreatePost/index.js

```diff
@@ -366,7 +366,6 @@ const CreatePost = () => {
       );
     }
   };
-
   const takePhoto = async () => {
     const {success, message} = await requestCameraPermission();
     if (success) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New lines have been added before and after the function `takePhoto`. This doesn't affect the functionality but improves readability by separating logical blocks of code.
```diff
+ 
+ const takePhoto = async () => {
+   const {success, message} = await requestCameraPermission();
+   if (success) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->